client/migrate/src/main.rs

```diff
@@ -0,0 +1,41 @@
+#![no_std]
```

Comment:
add a readme for this so that people understand what it is meant for / why they do or do not care